src/api/subscriptions.js

```diff
@@ -171,6 +171,7 @@ export const SubscriptionFlows = {
  * @typedef {{
  *   skus: (!Array<string>|undefined),
  *   list: (string|undefined),
+ *   isClosable: (boolean|undefined),
```

Comment:
The `isClosable` field only applies to one of three APIs: `showOffers`. Do you think it's confusing to keep it in this structure? Generally, I like this idea, but still wondering... At the very least we should document that it only applies to `showOffers`. Or alternatively, we could extend `isClosable` to all offer-based APIs. WDYT?

Reply:
I think, keeping `isClosable` for all offersView would be better since, in future if required this feature would be available to make `AbbrvOfferView` and `OptionsView` non-closable.